test/test-signal.js

```diff
@@ -0,0 +1,64 @@
+'use strict';
+
+// Testcase to produce NodeReport on signal
```

Comment:
I'd be more clear, "on signal interrupting a js busy-loop, showing it is interruptible"